ui/src/components/policy/PolicyRow.js

```diff
@@ -394,7 +395,15 @@ export class PolicyRow extends React.Component {
         this.setState({ [key]: value });
     }
 
+    onClickFunction(route) {
+        this.props.router.push(route, route);
```

Comment:
can you help me understand this route push x2 ?

Reply:
I really don't know why but it works this way I looked on other pages